src/Authentication/Authentication.Core/Interfaces/IGraphOptions.cs

```diff
@@ -2,14 +2,11 @@
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
 
-using System;
-using System.Security;
-using System.Security.Cryptography.X509Certificates;
-
 namespace Microsoft.Graph.PowerShell.Authentication
 {
     public interface IGraphOption
     {
         bool EnableWAMForMSGraph { get; set; }
+        bool EnableATPoPForMSGraph { get; set; }
```

Comment:
Strictly speaking, adding a property to an interface is a breaking change. Data objects like this should not be backed by interfaces.

Reply:
@peombwa Thoughts on this comment?

Reply:
I don't think these settings should be part of an interface that has the very generic name IGraphOption.  All other authentication parameters are provided as parameters to the Connect-MgGraph. I don't understand why these properties should be handled differently.

Reply:
They are part of the interface which allows the user customize their msgraph ps experience. This is related to the Set-MgGraphOption and Get-MgGraphOption commands